composer.json

```diff
@@ -4,15 +4,15 @@
     "description": "Builds nice branch names.",
     "license": "proprietary",
     "require": {
-        "php": ">=7.3",
+        "php": ">=8.0",
         "chobie/jira-api-restclient": "^2.0@dev",
         "cocur/slugify": "^4.0",
         "laminas/laminas-filter": "^2.9",
         "symfony-cmf/slugifier-api": "^2.0",
-        "symfony/console": "^5.0",
-        "symfony/options-resolver": "^5.0",
-        "symfony/process": "^5.0",
-        "symfony/yaml": "^5.0"
+        "symfony/console": "^5.0 | ^6.0",
```

Comment:
U can drop v5

Reply:
it can be blocker if some one use this tool.